blocks/community-cards/community-cards.css

```diff
@@ -0,0 +1,162 @@
+.community-cards .cc-container {
+  gap: 2rem;
+  color: var(--text-color);
+  display: flex;
+  overflow-x: scroll;
+  font-size: var(--body-font-size-xxs);
+}
+
+.community-cards .cc-container::-webkit-scrollbar {
+  display: none;
+}
+
+.community-cards .cc-container .card {
+  cursor: default;
+  background: var(--body-text-color) 0 0 no-repeat padding-box;
+  box-shadow: 0 3px 6px #000029;
+  border-radius: 16px;
+  opacity: 1;
+  flex: 1;
+  min-width: 240px;
+  overflow: none;
+  padding: 1.5rem 0;
+  position: relative;
+}
+
+.community-cards .cc-container .card .card-header{
```

Comment:
Small suggestion, can we have single space before '{' brace, ? so, it will be '.community-cards .cc-container .card .card-header {'. 

Doo it for all selectors.